src/icon-button.jsx

```diff
@@ -28,6 +32,26 @@ const IconButton = React.createClass({
     };
   },
 
+  statics: {
+    getRelevantContextKeys(muiTheme) {
```

Comment:
and what about `https://github.com/callemall/material-ui/blob/master/src/buttons/flat-button-label.jsx#L44` that doesn't use any argument?